src/UI/HTMLTooltip.js

```diff
@@ -84,6 +86,14 @@ export class HTMLTooltip {
         document.body.appendChild(this.host)
     }
     remove () {
+        // Reset input icon styles back to initial
+        const form = this.device?.activeForm
+        const input = form?.activeInput
+        if (input) {
+            const initialStyles = getIconStylesBase(input, form)
+            addInlineStyles(input, initialStyles)
+        }
+
```

Comment:
I wonder whether this should rather live within the form class itself, since we're decorating the input from there. What do you think?

Reply:
@GioSensation Ideally, but the form doesn't control all occurrences of when the tooltip closes, right?
e.g. clicking away from the tooltip is set up in `HTMLTooltipUIController.js` in the `_pointerDownListener`
Can you think of a way to wrangle this?